schemas/telemetry/focus-event/focus-event.1.schema.json

```diff
@@ -24,11 +24,11 @@
             "type": "string"
           },
           {
-            "type": "string"
+            "type": [ "string", "null" ]
           },
           {
             "additionalProperties": {
-              "type": "string"
+              "type": [ "string", "null" ]
```

Comment:
The validation breaks on the example ping if I remove this. I believe it's indicating the value is NULL, not the key.